domains/ashishagr.json

```
@@ -17,9 +19,6 @@
           "target": "mx3.zoho.com",
           "priority": 50
         }],
```

you forgot the excess comma
```suggestion
        }]
```